install_spot_ros2_arm.sh

```diff
@@ -0,0 +1,9 @@
+ROS_DISTRO=humble  # only humble is currently supported by install script
+# Install BD API
+pip3 install bosdyn-client bosdyn-mission bosdyn-api bosdyn-core
+# Install ROS dependencies
+sudo apt install -y ros-$ROS_DISTRO-joint-state-publisher-gui ros-$ROS_DISTRO-tf-transformations ros-$ROS_DISTRO-xacro ros-$ROS_DISTRO-depth-image-proc
+# Install bosdyn_msgs - automatic conversions of BD protobufs to ROS messages
+wget -q -O /tmp/ros-humble-bosdyn-msgs_3.2.0-0jammy_arm64.deb https://github.com/bdaiinstitute/bosdyn_msgs/releases/download/bosdyn_msgs-v.3.2.0-humble-arm64/ros-humble-bosdyn-msgs_3.2.0-0jammy_arm64.deb
```

Comment:
Whenever versions are hardcoded it always makes me nervous that we'll forget one in a future update, though the impact isn't too severe.

Maybe (in a future followup - feel free to assign to me), we can make the script detect the architecture and consolidate these scripts - not needed now.